fastai/layer_optimizer.py

```diff
@@ -38,10 +38,18 @@ def set_lrs(self, lrs):
         set_lrs(self.opt, lrs)
         self.lrs=lrs
 
+    def set_wds_out(self, wds):
+        if not isinstance(wds, Iterable): wds=[wds]
```

Comment:
Maybe instead of

```
if not isinstance(wds, Iterable): wds=[wds]
if len(wds)==1: wds=wds*len(self.layer_groups)
```

you could make a little function:

```
wds = listify(wds, self.layer_groups)
```

...where the second param is used directly if it's an int, or we use its `len` if it's listy? I think we've used this pattern in quite a few places.

Reply:
Yeah, it's used quite a few times and you're right, a helper function would make the code cleaner.